docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx

```diff
@@ -10,7 +10,7 @@ In this guide, we'll walk you through creating and revoking user and agent token
 
 ---
 
-## Managing agent tokens
```

Comment:
This should remain `Managing`, as the section now includes instructions for two (or three, see my other comment) different things: creating the token, assigning permissions, and revoking it.

We could add a heading to line 22 (I can't comment directly on it, apparently) for token creation: `### Creating agent tokens`